Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -532,8 +532,11 @@ def audit_conflicts
       end
     end
 
+    versioned_conflicts_whitelist = %w[node@ bash-completion@]
```

Comment:
I assume you mean freezing it to make it an immutable array, not making it a true constant.